### DISCUSSION TITLE
[Feature] Allow selection of a whole series of search results with shift-click

### DISCUSSION THREAD

**Discussion by bhugh** [immich-app/immich#7742] [Feature Request]
### The feature

In face search and most other screens where an array of photos is presented, you can select individual photos by clicking each photo, or select all via the button at the time.

But in addition, you can select a whole series of photos by selecting one photo, then shift-clicking the second photo.  This selects all photos between the two.  Also, you can click and shift-click again to add a second sequence of photos to the selection.

It would be very useful to be able to do this in search results, too.  Right now, the only choices are click each photo individually, or click the button up top to select all.

Thanks for making a great app!

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #6162
